### ISSUE TITLE
Hide to tray via Esc or something

### ISSUE THREAD

**Issue by sergeevabc** [bailey27/cppcryptfs#122]
Dear @bailey27,
Could you be so kind to make GUI part of `cppcryptfs` a bit more friendly towards oldschool keyboard users?
After a correct password is entered and a drive is successfully mounted, I’d like to hide the app into tray, but
neither Esc, nor Alt/Ctrl+F4 works.

**Comment by bailey27** [comment]
Thanks for pointing this out.  I'll try to make Alt+F4 do that.

**Comment by bailey27** [comment]
I want be able to either minimize to task bar or minimize to tray without having to have a setting to control what the minimize action does.

So I chose to make what actions would normally minimize the program minimize it to task bar and what would normally exit it minimize to tray.

Because Alt+F4 normally has the same effect as clicking on the X in the upper right corner, I want it to perform the same action (which in the case of cppcryptfs is minimize to tray).

You can still exit with keyboard with Alt+X (the shortcut for exit) if you are on the mount tab.

**Comment by bailey27** [comment]
I changed my mind and wanted to use Alt+T to minimize to tray.

I hoped to address this with the release I did today.   However, what I thought would work (using keyboard accelerator resource) wasn't working.  I wanted to get the release out to address the ejecting USB drives issue, and I also wanted to build against the latest Dokany that was released a few days ago.

I hope to get this done in the near future though.
